assembl/static2/js/app/client.js

```diff
@@ -40,16 +46,28 @@ const networkInterface = createNetworkInterface({
 networkInterface.use([
   {
     applyMiddleware: function (req, next) {
-      Sentry.addBreadcrumb({
-        category: 'graphql',
-        message: `GraphQL operation: ${req.request.operationName}`,
-        data: {
-          variables: req.request.variables
-        },
-        level: 'info'
-      });
+      if (!req.options.headers) {
+        req.options.headers = {}; // Create the header object if needed.
+      }
+      co(async () => {
+        if (useCSRFProtection === 'true') {
+          const responseToken = await getCSRFToken();
+          if (responseToken.text === 'ok') {
+            req.options.headers['X-XSRF-TOKEN'] = Cookies.get('_csrf');
+          }
+        }
 
-      next();
+        Sentry.addBreadcrumb({
```

Comment:
Nah, keep the `next()` above, and put Sentry in the next index position. They are different middlewares, CSRF and Sentry, but right now, they're both one big middleware.

Reply:
Didn't realise I had removed this!

Reply:
I moved it to below the sentry - you sure?

Reply:
Yeah, it's a middleware array. You're adding another middleware, so you want your middleware func to follow the pattern

Reply:
OK